setup.py

```diff
@@ -65,7 +65,14 @@ def load_requirements(*requirements_paths):
         'workbench',
     ],
     install_requires=load_requirements('requirements.txt'),
-    tests_require=load_requirements('test-requirements.txt'),
+    tests_require=[
```

Comment:
Ugh.. this was one of those "last-minute, what could it hurt" sort of changes, which, well... never end up being so straight-foward.  There are some additional caveats to this change, namely, `tests_require` doesn't actually install the packages into the `virtualenv`; it just installs them while running the tests.

While I'd like to revisit this in the future, I want to avoid rabbit-holing until I can devote time to straightening out all of the plumbing.

For the sake of this PR, I have reverted the change.